iwa-console/MSAL/MSALClientHelper.cs

```diff
@@ -0,0 +1,270 @@
+﻿using Microsoft.Identity.Client;
+using Microsoft.Identity.Client.Broker;
+using Microsoft.Identity.Client.Extensions.Msal;
+using Microsoft.IdentityModel.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace iwa_console.MSAL
+{
+    /// <summary>
+    /// Contains methods that initialize and use the MSAL SDK
+    /// </summary>
+    /// <autogeneratedoc />
+    public class MSALClientHelper
+    {
+        /// <summary>
+        /// As for the Tenant, you can use a name as obtained from the azure portal, e.g. kko365.onmicrosoft.com"
+        /// </summary>
+        public readonly AzureADConfig AzureADConfig;
+
+        /// <summary>
+        /// Gets the authentication result (if available) from MSAL's various operations.
+        /// </summary>
+        /// <value>
+        /// The authentication result.
+        /// </value>
+        public AuthenticationResult AuthResult { get; private set; }
+
+        /// <summary>
+        /// Gets a value indicating whether this instance of PublicClientApp was initialized with a broker .
+        /// </summary>
+        /// <value>
+        ///   <c>true</c> if this instance is broker initialized; otherwise, <c>false</c>.
+        /// </value>
+        public bool IsBrokerInitialized { get; private set; }
```

Comment:
I don't think we should expose this to end-users

Reply:
That can be refactored but are we supporting Brokers for `MacOS` and `Linux`?

Reply:
No, but MSAL will fallback to a browser on Mac / Linux automatically. App developer doesn't need to worry about this.